### PR TITLE
Finish ConnectDataWithAPI

### PR DESCRIPTION
**Pull request by Sebastian-Basmachi**

### CODE REVIEW

File: src/app/actions/index.ts [hunk 1]
```diff
@@ -0,0 +1,19 @@
+import axios from 'axios';
+import {INFO_API} from './constants';
+
+const urlApi: string = 'https://data.messari.io/api/v1/assets';
+
+export const getInfoApi = () => {
```
Comment (tomasdematos) [inline src/app/actions/index.ts]: only getInfo

File: src/components/cryptocurrenciesList/index.tsx [hunk 2]
```diff
@@ -1,36 +1,28 @@
...
-
+  const dispatch = useDispatch();
   const renderItem = ({item}: {item: Crypto}) => <List item={item} />;
+  const infoApi = useSelector(state => state.allCryptos);
+
+  useEffect(() => {
+    dispatch(getInfoApi());
+  }, [dispatch]);
+
+  const cryptoSelected = ['BTC', 'ETH', 'XRP'];
```
Comment (tomasdematos) [inline src/components/cryptocurrenciesList/index.tsx:19]: no default selected cryptos

Comment (tomasdematos) [reply src/components/cryptocurrenciesList/index.tsx:19]: the validation below should be unnecessary then

Comment (Sebastian-Basmachi) [reply src/components/cryptocurrenciesList/index.tsx:19]: I know but it was to demonstrate that works and to do the video, in the next task the user will can select the cryptos

**Pull request merged**